Formula/neovim.rb

```diff
@@ -0,0 +1,72 @@
+class Neovim < Formula
+  desc "Ambitious Vim-fork focused on extensibility and agility"
+  homepage "https://neovim.io"
+
+  stable do
+    url "https://github.com/neovim/neovim/archive/v0.2.0.tar.gz"
```

Comment:
If there aren't resources or dependencies specific to stable or head, you can make this top section
```
  url "https://github.com/neovim/neovim/archive/v0.2.0.tar.gz"
  sha256 "72e263f9d23fe60403d53a52d4c95026b0be428c1b9c02b80ab55166ea3f62b5"
  head "https://github.com/neovim/neovim.git", :shallow => false
```